spec/controllers/admin/accounts_controller_spec.rb

```diff
@@ -307,5 +307,32 @@ def self.name
         expect { delete :destroy, id: account }.not_to change { Pageflow::Account.count }
       end
     end
+
+    describe '#confirm_invitations' do
+      it 'redirects with warning when there is no invitation for the user/account' do
+        account = create(:account)
+        user = create(:user)
+
+        sign_in(user)
+        request.env['HTTP_REFERER'] = admin_users_path
+
+        get(:confirm_invitations, id: account.id)
+
+        expect(flash[:alert]).to eq(I18n.t('pageflow.admin.users.no_invitations_available'))
+      end
+
+      it 'redirects with confirmation message that the invitations were turned into memberships' do
+        account = create(:account)
+        user = create(:user)
+        create(:invitation, entity: account, user: user)
+
+        sign_in(user)
+        request.env['HTTP_REFERER'] = admin_users_path
+
+        get(:confirm_invitations, id: account.id)
+
+        expect(flash[:notice]).to eq(I18n.t('pageflow.admin.users.invitations_confirmed'))
+      end
+    end
```

Comment:
Let's test that this action
* only confirms invitations of the current user and leaves alone invitations to the same account for other users 
* only confirms invitations to the given account if the user has pending invitations for other accounts